components/brave_webtorrent/stories/components/torrentFileList.tsx

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { Heading, Table } from 'brave-ui/components'
+import { Cell, Row } from 'brave-ui/components/dataTables/table/index'
+import { LoaderIcon } from 'brave-ui/components/icons'
+import * as prettierBytes from 'prettier-bytes'
+
+// Constants
+import { File, TorrentObj } from '../../extension/constants/webtorrentState'
+
+// Locale
+import locale from '../fake-locale';
```

Comment:
...oh, like this :-)

Reply:
Was thinking since there is quite a bit of verbiage updates between current and new ui screens, could start establishing the locale in storybook and then maybe translate over when it's in a good place? We'd want to do this anyway for translations right?

Reply:
well, whenever the actual UI needs the strings then implement them yeah. I just like that we're using an individual locale file for these screens instead of one that contains rewards / ntp / webtorrent combined.